src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -145,12 +180,39 @@ export class QuickBar extends LitElement {
     this._activatedIndex = 0;
   }
 
+  private _handleSelected(ev: SingleSelectedEvent) {
+    const index = ev.detail.index;
+    const item = (ev.target as any).items[index].item;
+    this.processItemAndCloseDialog(item, index);
+  }
```

Comment:
Is the purpose of this function just to separate the concept of "handling a selection" and "how to execute quick bar commands"?

Reply:
it is to get the index and items from the event detail

Reply:
But that was already being done inside processItemAndCloseDialog. Seems like an unnecessary abstraction to me.